h2o-genmodel-extensions/deepwater/src/main/java/hex/genmodel/algos/deepwater/DWImageConverter.java

```diff
@@ -2,8 +2,8 @@
 
 import hex.genmodel.GenModel;
 import hex.genmodel.easy.CategoricalEncoder;
+import hex.genmodel.easy.DefaultRowToRawDataConverter;
```

Comment:
the changes in non-TE-related genmodel classes are due to renamings:
- I mentioned `CategoricalEncoding` enum -> `CategoricalEncodings`.
- here, `RowToRawDataConverter` has been renamed to `DefaultRowToRawDataConverter` and the extracted interface keeps the oiginal name.